dom.go

```diff
@@ -418,20 +454,26 @@ func renderComponent(comp Component, prevRender *HTML) (h *HTML, skip bool) {
 		nextRender = Tag("noscript")
 	}
 
-	// Restore the actual rendered HTML.
+	// Reconcile the actual rendered HTML.
 	nextRender.reconcile(prevRender)
 
 	// Update the context to consider this render.
-	comp.Context().prevRender = nextRender
-	comp.Context().prevComponent = comp
-	comp.Context().prevRenderComponent = doCopy(comp)
+	//
+	// TODO: In the event that this really is the previous component instance,
```

Comment:
I suspect the behaviour that the comment refers to is a bug I'm already aware of, where a rerender gets queued for a child component that in the mean time gets removed or replaced in the DOM by a parent rerender.  If this is indeed the same bug, it will be fixed by lifecycle event changes (mount/unmount) that I already have in the pipeline.